src/validate.ts

```diff
@@ -0,0 +1,75 @@
+
+'use-strict';
+
+
+function generateRequestID(min, max){
+        return Math.floor(Math.random() * (max-min) + min).toString()
+}
+    
+
+export function generatSyncRequest(){
+    const requestId = generateRequestID(100,999);
+    return {
+        requestId,
+        inputs: [{
+          intent: "action.devices.SYNC"
+        }]
+    }
+}
+
+
+let deviceIDs: Array<string>;
```

Comment:
Do we need this array for anything? I think it can be removed.

Reply:
This was just defining the type for the device ID so that if another type was there it would throw an error